.github/workflows/llvm.yml

```diff
@@ -30,8 +34,22 @@ jobs:
           key: ${{ steps.cache-key.outputs.cache-key }}
           lookup-only: true
 
-      - name: Install Tools
+      - name: Download LLVM from Rust CI
+        id: download-llvm
         if: steps.cache-llvm.outputs.cache-hit != 'true'
+        run:
+          wget https://ci-artifacts.rust-lang.org/rustc-builds/${{ steps.ls-remote.outputs.sha }}/rust-dev-nightly-${{ matrix.target }}.tar.xz && \
+            echo "download-llvm=success" << "$GITHUB_OUTPUT" || \
```

Comment:
these shovels are pointing the wrong way, but also this isn't the right way to do this.

instead we should check `steps.download-llvm.outcome == 'success'` below; see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/contexts#steps-context